player_test.py

```diff
@@ -154,11 +154,14 @@ def test_returns_user_input_for_input_is_red(self):
     prompt_text = "please enter your choice of ant"
     expected_result = "red"
     input_patcher = mock.patch('builtins.input', return_value = "red")
+    print_patcher = mock.patch('builtins.print')
     input_mock = input_patcher.start()
+    print_mock = print_patcher.start()
```

Comment:
I've not seen a test to assert that the allowed choices are printed. I suggest that this is covered.